docs/product_search/index.md

```diff
@@ -3,4 +3,82 @@ layout: default
 title: Product Search
 has_children: true
 nav_order: 8
+has_toc: false
 ---
+
+# Product Search
+Product search is based on query parameters that determine which products are returned and in which order. In addition to the search query itself, there are parameters that allow you to customize the search further, for example you could:
+
+- Choose whether unavailable products are returned
```

Comment:
I actually don't know what unavailable means here. Can we define what gets returned somewhere here? My guess is product_search by default returns all published&'settings>available' products with at least 1 not-hidden and not-sold out variant? 

include_sold_out includes published products with at least 1 not-hidden variant (as above but includes sold out)

Reply:
Sorry I meant sold out by this - but the available vs unavailable etc is a good thing to clarify - will add